pkg/apis/batch/v1/defaults.go

```diff
@@ -39,4 +39,12 @@ func SetDefaults_Job(obj *Job) {
 		obj.Spec.Parallelism = new(int32)
 		*obj.Spec.Parallelism = 1
 	}
+	// When manualSelector is true, set metadata.Labels according to
```

Comment:
If you get a chance before the bot merges it, fix the comment. In `batch/v1` there's no `manualSelector`, only automatic one. Generally no comment here is needed.